gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -347,43 +347,47 @@ message VehiclePosition {
   }
   optional CongestionLevel congestion_level = 6;
 
-  // The degree of passenger occupancy of the vehicle. This field is still
-  // experimental, and subject to change. It may be formally adopted in the
-  // future.
+  // The degree of passenger occupancy of the vehicle
   enum OccupancyStatus {
     // The vehicle is considered empty by most measures, and has few or no
     // passengers onboard, but is still accepting passengers.
-    EMPTY = 0;
+    EMPTY = 0 [deprecated=true];
 
-    // The vehicle has a relatively large percentage of seats available.
-    // What percentage of free seats out of the total seats available is to be
-    // considered large enough to fall into this category is determined at the
-    // discretion of the producer.
-    MANY_SEATS_AVAILABLE = 1;
+    // The vehicle has seats available.
+    SEATS_AVAILABLE = 1;
 
     // The vehicle has a relatively small percentage of seats available.
     // What percentage of free seats out of the total seats available is to be
     // considered small enough to fall into this category is determined at the
     // discretion of the feed producer.
-    FEW_SEATS_AVAILABLE = 2;
+    // Deprecated, if you need more granularity than the available enum please use occupancy_percentage
+    FEW_SEATS_AVAILABLE = 2 [deprecated=true];
 
     // The vehicle can currently accommodate only standing passengers.
     STANDING_ROOM_ONLY = 3;
```

Comment:
If we're moving to only 3 enum values, would it make sense to just adopt the same terminology as SIRI?

That would mean changing `STANDING_ROOM_ONLY` to `STANDING_AVAILABLE` - the semantics are close, but I'm not sure they are the same.

Reply:
Make sense to me 👍